### PR TITLE
Fix monitoring.sh flake (hopefully)

### PR DESCRIPTION
**Pull request by zmerlynn**
This was staring at me yesterday, and I even commented that "huh,
there's got to be something wrong with the firewall rules", but then
job/kubernetes-e2e-gce/1002/tapResults/ made it obvious: If you have
two e2e jobs running at the same time in the same project (hint:
Jenkins does), they'll race with each other, since resource names are
project scoped.

### CODE REVIEW

File: hack/e2e-suite/monitoring.sh [hunk 1]
```diff
@@ -28,20 +28,19 @@ source "${KUBE_ROOT}/cluster/$KUBERNETES_PROVIDER/util.sh"
 
 MONITORING="${KUBE_ROOT}/cluster/addons/cluster-monitoring"
 KUBECTL="${KUBE_ROOT}/cluster/kubectl.sh"
-MONITORING_FIREWALL_RULE="monitoring-test"
+BIGRAND=$(printf "%x\n" $(( $RANDOM << 16 | $RANDOM ))) # random 2^32 in hex
```
Comment (lavalamp) [inline hack/e2e-suite/monitoring.sh]: What sets $RANDOM?

**Comment by lavalamp** [comment]
LGTM w/ answer to question

File: hack/e2e-suite/monitoring.sh (hunk 1 above)
Comment (zmerlynn) [inline hack/e2e-suite/monitoring.sh]: bash :)

Comment (zmerlynn) [inline hack/e2e-suite/monitoring.sh]: http://tldp.org/LDP/abs/html/randomvar.html

**Pull request merged**